### DISCUSSION TITLE
[Fixed] Integrations page showing a 404

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#221] [Help] [answered]
Hi,

Just to let you know that integration links in https://vercel.com/integrations are dead.

For example with Checkly Integration: go to https://vercel.com/integrations and search for Checkly, click on the Checkly card pointing to https://vercel.com/integrations/checkly and you see a _404 - Page Not Found_ error.

Cheers,
Clement

**Comment by leerob** [comment, marked as answer]
Thanks for letting us know, this has been fixed!
